bitnami/thanos/values.yaml

```diff
@@ -61,16 +61,17 @@ querier:
   ##
   replicaLabel: replica
   
-  ## Dinamically configure store APIs using DNS discovery
+  ## Dynamically configure store APIs using DNS discovery
```

Comment:
Please, could you add a small phrase to the `README.md` at the values section explaining how it is going to work

Reply:
Will do that. Thx